tests/sync/test_rest_api.py

```
@@ -242,10 +242,29 @@ async def test_denial_end_to_end(self):
         try:
             while True:
                 response = await seb_communicator.receive_from()
-        except:
-            pass
+        except asyncio.CancelledError as cancel_err:
+            exc_type, exc_value, exc_tb = sys.exc_info()
```

Ruff (0.11.9)

Undefined name `sys`

(F821)